vali_objects/position.py

```diff
@@ -93,14 +88,7 @@ def get_cumulative_leverage(self) -> float:
                 break
 
             prev_leverage = current_leverage
-
-            # Clamp
-            if current_leverage + order.leverage > self.trade_pair.max_leverage:
-                current_leverage = self.trade_pair.max_leverage
-            elif current_leverage + order.leverage < -self.trade_pair.max_leverage:
-                current_leverage = -self.trade_pair.max_leverage
-            else:
-                current_leverage += order.leverage
```

Comment:
clamping only needs to be done on initial order processing. remove it here as it is redundant.